exe/main.cc

```diff
@@ -41,7 +40,20 @@ int startAgent(std::vector<std::string> arguments) {
     }
 #endif
 
+    if (!Configuration::Instance().isInitialized()) {
+        std::thread idle_thread { []() {
```

Comment:
I mean, about what we're doing with the dumb thread when Configuration is not initialized.